Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:4.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MapboxDirections",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "MapboxDirections",
+            targets: ["MapboxDirections"]),
+        ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/raphaelmor/Polyline.git", .branch("master"))
```

Comment:
@1ec5 Would you mind making a new release of Polyline?

Changes since [v4.2.0](https://github.com/raphaelmor/Polyline/releases/tag/v4.2.0):
 Added support for Swift Package Manager ([#51](https://github.com/raphaelmor/Polyline/pull/51))

Reply:
[All set](https://github.com/raphaelmor/Polyline/releases/tag/v4.2.1).